### PR TITLE
Verbose page cache flush tracer

### PR DESCRIPTION
**Pull request by MishaDemianenko**
Introduce verbose page cache flush tracer that will
log page cache events that happen during page cache flush together with
some additional information: flush speed, number of pages affected by flush
etc.
Useful for investigation purposes when there is a need to see how much pages
affected, how much data is flushed, how fast flush is actually progress.

### CODE REVIEW

File: enterprise/kernel/src/main/java/org/neo4j/kernel/monitoring/tracing/VerboseTracerFactory.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+import org.neo4j.helpers.Service;
+import org.neo4j.io.pagecache.tracing.PageCacheTracer;
+import org.neo4j.kernel.impl.util.JobScheduler;
+import org.neo4j.kernel.monitoring.Monitors;
+import org.neo4j.logging.Log;
+import org.neo4j.time.SystemNanoClock;
+
+@Service.Implementation( TracerFactory.class )
+public class VerboseTracerFactory extends DefaultTracerFactory
```
Comment (chrisvest) [inline enterprise/kernel/src/main/java/org/neo4j/kernel/monitoring/tracing/VerboseTracerFactory.java:30]: This won't be available unless you add it to a META-INF/services file.

Comment (MishaDemianenko) [reply enterprise/kernel/src/main/java/org/neo4j/kernel/monitoring/tracing/VerboseTracerFactory.java:30]: unless you rely on ServiceProcessor or any other auto generator to do that automatically:) he

Comment (chrisvest) [reply enterprise/kernel/src/main/java/org/neo4j/kernel/monitoring/tracing/VerboseTracerFactory.java:30]: Are you sure that's still plugged in and running?

Comment (MishaDemianenko) [reply enterprise/kernel/src/main/java/org/neo4j/kernel/monitoring/tracing/VerboseTracerFactory.java:30]: yep, will gonna add integration test that uses verbose tracer

**Comment by MishaDemianenko** [comment]
@chrisvest integration test for newly introduced tracer added

**Review by chrisvest — APPROVED** [review]

**Pull request merged**